src/vm/wren_compiler.c

```diff
@@ -3215,6 +3234,9 @@ static void classDefinition(Compiler* compiler, bool isForeign)
   classVariable.scope = compiler->scopeDepth == -1 ? SCOPE_MODULE : SCOPE_LOCAL;
   classVariable.index = declareNamedVariable(compiler);
   
+  ObjString* className = AS_STRING(wrenNewString(compiler->parser->vm, 
+    compiler->parser->previous.start, compiler->parser->previous.length));
```

Comment:
Yeah, I was thinking that too, but I figured it was safest to just make a separate string since I don't yet fully understand what happens with that value in the VM.

Can you provide a little more detail on exactly what would happen so I can comment it accurately?